sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentOptions.java

```diff
@@ -70,4 +71,27 @@ public AnalyzeSentimentOptions setIncludeOpinionMining(boolean includeOpinionMin
         this.includeOpinionMining = includeOpinionMining;
         return this;
     }
+
+    /**
+     * Get the value of {@code stringIndexType}.
+     *
+     * @return The value of {@code stringIndexType}.
+     */
+    public StringIndexType getStringIndexType() {
+        return stringIndexType;
+    }
+
+    /**
+     * Set the value of {@code stringIndexType}. It could be 'TextElements_v8', 'UNICODE_CODE_POINT', or
```

Comment:
nit: Do we want to keep mentioning all the values here, the list could get long?

Reply:
I can remove it.